spec/spec.md

```diff
@@ -83,11 +83,61 @@ on GitHub (see above) and other mediums (e.g. DIF) where this work is being done
 
 ## Linked Verifiable Presentation Service Endpoint
 
-TODO
+To enable discovery of linked verifiable presentations from the resolution of a DID, there must exist a DID Document
+mechanism for expressing locations where Verifiable Presentations MAY be located. To that end, the following section
+codifies the `LinkedVerifiablePresentations` Service Endpoint, which allows a DID controller to specify a list of
+presentations over which they assert control. Presentations asserted within the `LinkedVerifiablePresentations` endpoint
+descriptor can then be subsequently crawled by verifying parties to locate and verify any Verifiable Presentation
+resources that may exist.
+
+### Example
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://identity.foundation/linked-vp/v1"
+  ],
+  "id": "did:example:123",
+  "verificationMethod": [{
+    "id": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
+    "type": "JsonWebKey2020",
+    "controller": "did:example:123",
+    "publicKeyJwk": {
+      "kty": "OKP",
+      "crv": "Ed25519",
+      "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ"
+    }
+  }],
+  "service": [
+    {
+      "id": "did:example:123#foo",
+      "type": "LinkedVerifiablePresentations",
+      "serviceEndpoint": {
+        "presentations": [
+          "https://foo.example.com/verifiable-presentation.jsonld",
+          "https://identity.foundation/presentation.json"
+        ]
+      }
+    },
+    {
+      "id": "did:example:123#bar",
+      "type": "LinkedVerifiablePresentations",
+      "serviceEndpoint": "https://bar.example.com/verifiable-presentation.jwt"
+    }
+  ]
+}
+```
 
 ### Linked Verifiable Presentations
 
-TODO
+`LinkedVerifiablePresentations` endpoint descriptors are JSON objects composed as follows:
+
+- The object MUST contain an `id` property, and its value MUST be a valid DID URL reference.
+- The object MUST contain a `type` property, and its value MUST be the string "LinkedVerifiablePresentations".
+- The object MUST contain a `serviceEndpoint` property, and its value MUST be either a string or an object that contains
+  `presentations` as property, the value of which MUST be an array of one or more Uniform Resource Locators as described
```

Comment:
Would it make more sense to have `serviceEndpoint` be an array of presentation endpoints here instead of nesting it into the presentations `property`?

Reply:
Thank you. I picked up this pattern from other specs like https://identity.foundation/.well-known/resources/did-configuration/#linked-domain-service-endpoint. From the did-core spec `serviceEndpoint` could be all of the above: https://w3c.github.io/did-core/#services

I'm currently working on the security considerations and it's becoming clear to me that we probably want to have more attributes in the document than just the URLs.

I'll change the example to a list of strings so we can get it merged and I'll continue with the refinement of the data model in the security and privacy considerations PR.

Reply:
Done.

Reply:
sounds good looking forward to seeing the additions 👀